README.md

```diff
@@ -146,6 +146,11 @@ It is necessary to install the dependencies with:
 python3 -m pip install -e .[examples]
 ```
 
+### XGBoost
+
+If you wish to run the XGBoost notebook, you need the XGBoost library [dmlc/xgboost](https://github.com/dmlc/xgboost) available on your system.
+Using the geo engine backend with `pro` feature will provide the library. See [XGBoost: Installation](https://xgboost.readthedocs.io/en/stable/install.html) for installation instructions otherwise.
```

Comment:
"xgboost" feature as well

Reply:
I thought the `xgboost` feature is a subset of the `pro` feature, but it is actually extending the pro feature, isn't it?

https://github.com/geo-engine/geoengine/blob/8661625ed4f7920681e7a25e50d36902fe7f7fe2/operators/Cargo.toml#L14